File: FeedleDataTier/FeedleDataTier/Network/PostUserRequest.cs
using System.Text.Json.Serialization;
using FeedleDataTier.Models;

namespace FeedleDataTier.Network
{
    public class PostUserRequest : Request
    {
        [JsonPropertyName("user")]
        public User User { get; set; }

        public PostUserRequest(User user) : base(RequestType.PostUser)
        {
            this.User = user;
        }
    }
}

File: Backend/src/main/java/com/feedle/feedleapi/Networking/PostUserRequest.java
package com.feedle.feedleapi.Networking;

import com.feedle.feedleapi.Models.User;

public class PostUserRequest extends Request {
    private User user;

    public PostUserRequest(User user) {
        super(RequestType.PostUser);
        this.user = user;
    }

    public User getUser() {
        return user;
    }
}
